README.md

```diff
@@ -1,20 +1,29 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
-in a simple executable script/binary.
+Command-line tool and programmatic Python access to the Conjur API to manage your Conjur resources. 
+
+This repository includes both the self-contained Conjur command-line tool (`conjur`) and Python3-based SDK for Conjur.
 
 [![Test Coverage](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/test_coverage)](https://codeclimate.com/github/cyberark/conjur-api-python3/test_coverage) [![Maintainability](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/maintainability)](https://codeclimate.com/github/cyberark/conjur-api-python3/maintainability)
 
 ---
 
-### **Status**: Alpha
+## Certificate level
+
+### CLI
 
-#### **Warning: Naming and APIs are still subject to breaking changes!**
+![](https://img.shields.io/badge/Certification%20Level-Certified-6C757D?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
 
----
+The Conjur CLI is a **Certified** level project. It's been reviewed by CyberArk to verify that it will securely
+work with CyberArk DAP as documented. In addition, CyberArk offers Enterprise-level support for these features. For
+more detailed information on our certification levels, see [our community guidelines](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community).
 
-## Installing the code
+### SDK
+
+![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
+
+The Conjur Python SDK is a **Community** level project. It's a community contributed project that **is not reviewed or supported
+by CyberArk**. For more detailed information on our certification levels, see [our community guidelines](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community).
```

Comment:
below (line 32): Conjur OSS Suite (capital  S)
and line 33... בקיצור everwhere... Suite is with a capital S

(line 34) updgrade your Conjur version -> your Conjur OSS Suite version

Reply:
"when using integrations, choose the..."
change to "When (capital W). Full stop before it instead of semi-colon.

Reply:
choose the latest suite release that matches your Conjur **OSS** version.